src/toBeSorted/special_item_drop_mgr.cpp

```diff
@@ -366,34 +367,13 @@ const LowHealthReplacement LOW_HEALTH_REPLACEMENTS[] = {
      },
 };
 
-// TODO: Convert to enum once work on items has started
-static const u16 SPECIAL_ITEM_ARRAY[] = {
-    0,    // None
-    6,    // Heart
-    6,    // Heart
-    2,    // Green Rupee
-    3,    // Blue Rupee
-    4,    // Red Rupee
-    0x2b, // Farore Tear
-    0x28, // 5 Bombs
-    0x29, // 10 Bombs
-    7,    // Single Arrow
-    8,    // Bundle Arrows
-    0x39, // 5 Deku Seeds
-    0,    // None
-    0,    // None
-    0x1c, // ET Key Piece
-    0xa5, // Eldin Ore
-    58,   // ???
-    59,   // ???
-    183,  // ???
-    184,  // ???
-    185,  // ???
-    72,   // Fairy
-    72,   // Fairy
-    1,    // Small Key
-    34,   // Rupoor
-};
+int SPECIAL_ITEM_ARRAY[] = {ITEM_NONE,       ITEM_HEART,         ITEM_HEART,         ITEM_GREEN_RUPEE,
+                            ITEM_BLUE_RUPEE, ITEM_RED_RUPEE,     ITEM_FARORE_TEAR,   ITEM_5_BOMBS,
+                            ITEM_10_BOMBS,   ITEM_SINGLE_ARROW,  ITEM_BUNDLE_ARROWS, ITEM_5_DEKU_SEEDS,
+                            ITEM_NONE,       ITEM_NONE,          ITEM_KEY_PIECE,     ITEM_ELDIN_ORE,
+                            ITEM_UNK_58,     ITEM_UNK_59,        ITEM_UNK_183,       ITEM_UNK_184,
+                            ITEM_UNK_185,    ITEM_FAIRY_BUG_NET, ITEM_FAIRY_BUG_NET, ITEM_SMALL_KEY,
```

Comment:
```suggestion
                            ITEM_UNK_185,    ITEM_FAIRY, ITEM_FAIRY, ITEM_SMALL_KEY,
```

Cos the item name changed in the enum